yuque2markdown.py

```diff
@@ -111,6 +118,10 @@ def download_images_and_patch_html(output_dir_path, sanitized_title, html):
         no = 1
         for image in bs.find_all("img"):
             print("Download %s" % image["src"])
+            # check the url whether is valid, sometimes we import a wrong md so alert it to fix by handwrite.
+            if not is_valid_url(image["src"]):
+                print("\033[31mDocx '%s' has a wrong img path, check it!\033[0m" % sanitized_title)
```

Comment:
wrong -> invalid.

And please disable the Shell color, because some Shell env will not decode the prompts.